extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -372,9 +425,7 @@ export class AmpConsent extends AMP.BaseElement {
     this.consentStateManager_.getConsentInstanceState(instanceId)
         .then(state => {
           if (state == CONSENT_ITEM_STATE.UNKNOWN) {
```

Comment:
Can this be unknown because user dismissed? If so, do we always want to show it again?

Reply:
Right, there's this super rare condition that user trigger `postPromptUI` and dismiss before runtime figuring out the state and decide to ask for consent again. Another good point to have a CONSENT_UI_STATE : )